src/js/components/modals/ServiceScaleFormModal.js

```diff
@@ -65,8 +65,22 @@ class ServiceScaleFormModal extends mixin(StoreMixin) {
     this.props.onClose();
   }
 
-  onMarathonStoreServiceEditError({message:errorMsg}) {
+  onMarathonStoreServiceEditError({message:errorMsg = '', details: details}) {
     this.resetState();
+    let isErrorMsgToShort = errorMsg.toLowerCase() === 'object is not valid' &&
```

Comment:
I feel like this could be brittle checking for this exact string. What if we checked for `details` and tried to build the error from that first. If that doesn't exist, then we fall back to the generic `message` key? You guys might have more insight as to why we shouldn't do that though..?